src/master-playlist-controller.js

```diff
@@ -569,6 +569,23 @@ export class MasterPlaylistController extends videojs.EventTarget {
   }
 
   /**
+   * Convert the properties of an HLS track into an audioTrackKind.
+   *
+   * @private
+   */
+  audioTrackKind_(properties) {
+    let kind = properties.default ? 'main' : 'alternative';
+
+    if (properties.characteristics) {
+      if (properties.characteristics.indexOf('public.accessibility.describes-video') >= 0) {
```

Comment:
Sure - I wasn't sure how to deal with the line getting too long! I'll change that.